src/components/app/URLManager.js

```diff
@@ -1,91 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-// @flow
-
-import React, { PureComponent, PropTypes } from 'react';
-import { connect } from 'react-redux';
-import { getIsURLSetupDone } from '../../reducers/app';
```

Comment:
I removed all the logic about "urlSetupDone". As I understand it was there to make sure we `replaceState` a URL with an older format with a URL with the current format.

Instead I do it in https://github.com/devtools-html/perf.html/pull/550/files#diff-ecc5e5ed8ddbf07abeebb4d6dde28756R27, I believe this works the same.

(Note: I haven't removed all the logic from the reducers and the actions yet)